lib/ReactViews/Workbench/Controls/DimensionSelectorSection.tsx

```diff
@@ -38,6 +38,7 @@ class DimensionSelectorSection extends React.Component<PropsType> {
       dim =>
         !dim.disable &&
         isDefined(dim.options) &&
+        dim.options.length < 1000 &&
```

Comment:
could you put the 1000 into an exported const and use it in both files so that we don't accidently miss one of these if we ever need to change the limit?

Reply:
Yep good idea - done